lighthouse-core/report/handlebar-helpers.js

```diff
@@ -43,13 +47,13 @@ function calculateRating(value) {
 }
 
 /**
- * Figures out the total score for an aggregation
- * @param {{total: number}} aggregation
- * @return {number}
+ * Format number.
```

Comment:
> formatNumber formaters a number

I got less creative as I went along and just stole it from the also excellent [`format time`](https://github.com/GoogleChrome/lighthouse/blob/fe88c4ab830eff5182ac6754a30be353f89503d6/lighthouse-core/report/handlebar-helpers.js#L188)